.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@
 
 ## Does this affect other platforms - Amp, Apps, etc?
 
+## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
```

Comment:
Good shout on both counts, I have added comments to explain what paid content looks like and the link to that tag page so they can easily locate examples.